es6-classes/square.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) { super(); }
+  describe() {
+    return 'This is the width ' + this.width;
+  }
+}
+
+const square = new Square();
```

Comment:
When you instantiate `new Square()` you should be passing in an argument representing the `width`